democracy_club/apps/projects/templates/projects/cvs.html

```diff
@@ -1,30 +1,29 @@
 {% extends "projects/projects_base.html" %}
-{% load markdown_deux_tags %}
-
+{% load markdown_filter %}
 {% block title %}
-Democracy Club CVs
+    Democracy Club CVs
 {% endblock %}
 
 
 {% block content %}
-<div class="ds-stack-smaller">
-    <h1>Democracy Club CVs</h1>
-{% filter markdown %}
+    <div class="ds-stack-smaller">
```

Comment:
Are all these indents ok? Has that issue gone away with the new filter?

Reply:
No. Still indenting.